Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationIdentification.cshtml.cs

```diff
@@ -1,18 +1,51 @@
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
-namespace CO.CDP.OrganisationApp.Pages.SupplierRegistration
+namespace CO.CDP.OrganisationApp.Pages.Registration
 {
     public class OrganisationIdentificationModel : PageModel
     {
         private readonly ILogger<OrganisationIdentificationModel> _logger;
```

Comment:
Remove any unused objects, in this case logger.
We can always introduce later if required.